docs/07-BDD.md

```diff
@@ -1,90 +1,113 @@
+This is why it is common to talk about BDD in the context of a test frameworks (like Codeception).
+However, BDD goes much futher than just testing, and is more about managing the development process.
 
 ## What is Behavior Driven Development
 
-BDD was introduced by [Dan North](https://dannorth.net/introducing-bdd/). He described it as: 
+BDD was introduced by [Dan North](https://dannorth.net/introducing-bdd/). He described it as:
 
->  outside-in, pull-based, multiple-stakeholder, multiple-scale, high-automation, agile methodology. It describes a cycle of interactions with well-defined outputs, resulting in the delivery of working, tested software that matters.
+> outside-in, pull-based, multiple-stakeholder, multiple-scale, high-automation, agile methodology.
+It describes a cycle of interactions with well-defined outputs,
+resulting in the delivery of working, tested software that matters.
 
-BDD has its own evolution from the days it was born, started by replacing "test" to "should" in unit tests, and moving towards powerful tools like Cucumber and Behat, which made user stories (human readable text) to be executed as an acceptance test. 
+BDD has its own evolution from the days it was born, starting with replacing "test" to "should" in unit tests,
+and moving towards powerful tools like Cucumber and Behat,
+which made user stories (human readable text) executable as an acceptance test.
 
-The idea of story BDD can be narrowed to:
+The idea of BDD can be narrowed to:
 
 * describe features in a scenario with a formal text
 * use examples to make abstract things concrete
 * implement each step of a scenario for testing
 * write actual code implementing the feature
 
-By writing every feature in User Story format that is automatically executable as a test we ensure that: business, developers, QAs and managers are in the same boat.
+By writing every feature in a User Story format that is automatically executable as a test we ensure that:
+business, developers, QAs and managers are on the same page.
 
-BDD encourages exploration and debate in order to formalize the requirements and the features that needs to be implemented by requesting to write the User Stories in a way that everyone can understand.
+BDD encourages exploration and debate in order to formalize the requirements and the features
+that need to be implemented by requiring writing the User Stories in a way that everyone can understand.
 
-By making tests to be a part of User Story, BDD allows non-technical personnel to write (or edit) Acceptance tests.
+By making tests be a part of the User Story, BDD allows non-technical personnel to write (or edit) Acceptance tests.
 
-With this procedure we also ensure that everyone in a team knows what has been developed, what has not, what has been tested and what has not.
+With this procedure we also ensure that everyone in a team knows what has been developed,
```

Comment:
This also ensure that team mates know exactly what has been developed and tested so far.